package.json

```diff
@@ -75,7 +66,6 @@
     "conventional-changelog-cli": "^2.1.1",
     "copy-webpack-plugin": "^11.0.0",
     "cypress": "^12.11.0",
-    "cypress-image-snapshot": "^4.0.1",
```

Comment:
It seems to be used https://github.com/grafana/pyroscope/blob/main/og/cypress/support/commands.js#L26